test/se/vidstige/jadb/test/RealDeviceTestCases.java

```diff
@@ -23,10 +23,14 @@
     @BeforeClass
     public static void tryToStartAdbServer() {
         try {
+            new AdbServerLauncher().kill();
+        } catch (IOException | InterruptedException e) {
+            System.out.println("Could not kill adb-server");
+        }
```

Comment:
whats this? This is not part of starting adb server? Please instead add a new method called e.g. `tryRestartAdbServer`